### ISSUE TITLE
Suggestion regarding badges

### ISSUE THREAD

**Issue by Chaz6** [theodi/open-data-certificate#1699]
I would like to see a badge (of any type) require that the data be available without having to provide any personal information. I have seen some badges awarded where you must provide personal details such as name, organisation and email address as a prerequisite to obtaining the data and I do not feel that this is in the spirit of ODI.

The requirement for bronze states "BRONZE: data is openly licensed, available with no restrictions, accessible and legally reusable.". I would regard the providing of any data as a prerequisite for obtaining the data a restriction.

**Comment by olivierthereaux** [comment]
Good point. I understand that requiring contact information is to ensure that there is clear stewardship of the dataset and that the licensing is clear. @ldodds do you recall whether that was the rationale?
